doc/doxygen_manual.tex

```diff
@@ -123,6 +123,12 @@
 \tolerance=750
 \usepackage{etoc}
 \etocsettocstyle{\doxytocparskip}{\doxynormalparskip}
+\etocsetlevel{subsubsubsection}{4}
+\etocsetlevel{subsubsubsubsection}{5}
+\etocsetlevel{subsubsubsubsubsection}{6}
+\etocsetlevel{subsubsubsubsubsubsection}{7}
+\etocsetlevel{paragraph}{8}
+\etocsetlevel{subparagraph}{9}
```

Comment:
Actually this is not needed as etoc is left in "compatibility mode for line styles" i.e. there is no use of `\etocsetstyle`.  On the other hand, it does not hurt either.

Reply:
(and as you commented already, for this to work, upstream etoc currently at 1.2 needs an update)